### ISSUE TITLE
AVT 1 - Modal has DAP violation for missing accessible name

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2192]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Form Item Component
The following violations are reported:
An interactive element/widget must have an accessible name
[WAI-ARIA Specification for Widget Roles](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
[WAI-ARIA Specification for Accessible Name Computation](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_name)
<img width="1056" alt="Screen Shot 2019-04-10 at 2 30 08 PM" src="https://user-images.githubusercontent.com/21676914/55907839-6e4c5480-5b9d-11e9-9ffb-e941b2db2369.png">
